docs/source/software/basic_flight_controller.rst

```diff
@@ -1,3 +1,86 @@
 =======================
 Basic Flight Controller
 =======================
+
+The flight controller is responsible for passing high level position information to the px4. The Controller is written in C++ and designed to be extended for multiple use cases. To develop your own controller see the controller development page.
```

Comment:
Is Controller in the second sentence supposed to be capitalized? Not sure if this is representing the actual class, in which case maybe it should be code syntax (`Controller`).

Do we have a "controller development" page/section yet? If so, we should make that a hyperlink.

Reply:
no it shouldn't be capitalized, but we could call its something like `flightController` when we rename all the classes, which I think we really should do 

no we don't have that page, I was going to bring it up at today's meeting that we need a "controller development" and "troubleshooting" page.